Web/Edubase.Web.UI/Areas/Groups/Controllers/GroupController.cs

```diff
@@ -845,23 +846,49 @@ private async Task<GroupEditorViewModel> PopulateSelectLists(GroupEditorViewMode
-
-            viewModel.IsLocalAuthorityEditable = viewModel.GroupTypeId.OneOfThese(GT.ChildrensCentresCollaboration, GT.ChildrensCentresGroup)
-                                                 && viewModel.LinkedEstablishments.Establishments.Count == 0 && User.InRole(AuthorizedRoles.IsAdmin);
+            //this method only controls the visibility of the fields not access to the actual page
+            if (viewModel.GroupType.OneOfThese(GT.MultiacademyTrust, GT.SingleacademyTrust, GT.SchoolSponsor, GT.Federation)
+                && !viewModel.StatusId.OneOfThese(GS.CreatedInError, GS.Closed)
+                && User.InRole(AuthorizedRoles.IsAdmin))
+            {
+                viewModel.CanUserCloseAndMarkAsCreatedInError = true;
+            }
+            else if(viewModel.GroupType == GT.SecureSingleAcademyTrust
+                && !viewModel.StatusId.OneOfThese(GS.CreatedInError, GS.Closed)
+                && User.InRole(EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.YCS))
+            {
+                viewModel.CanUserCloseAndMarkAsCreatedInError = true;
+            }
 
+            if (viewModel.GroupTypeId.OneOfThese(GT.ChildrensCentresCollaboration, GT.ChildrensCentresGroup)
+                                                             && viewModel.LinkedEstablishments.Establishments.Count == 0 && User.InRole(AuthorizedRoles.IsAdmin))
+            {
+                viewModel.IsLocalAuthorityEditable = true;
+            }
 
-            if (User.InRole(AuthorizedRoles.CanBulkAssociateEstabs2Groups) && viewModel.GroupType.OneOfThese(GT.MultiacademyTrust, GT.SingleacademyTrust))
+            if (viewModel.GroupType.OneOfThese(GT.MultiacademyTrust, GT.SingleacademyTrust)
+                && User.InRole(AuthorizedRoles.CanBulkAssociateEstabs2Groups))
+            {
+                viewModel.CanUserEditClosedDate = true;
+                viewModel.CanUserEditStatus = true;
+                PopulateStatusSelectList(viewModel);
+            }
+            else if(viewModel.GroupType == GT.SecureSingleAcademyTrust
+                && User.InRole(EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.YCS))
             {
                 viewModel.CanUserEditClosedDate = true;
                 viewModel.CanUserEditStatus = true;
                 PopulateStatusSelectList(viewModel);
             }
 
-            if (User.InRole(AuthorizedRoles.IsAdmin) &&
-                viewModel.GroupType.OneOfThese(GT.MultiacademyTrust, GT.SingleacademyTrust))
+            if (viewModel.GroupType.OneOfThese(GT.MultiacademyTrust, GT.SingleacademyTrust)
+                && User.InRole(AuthorizedRoles.IsAdmin))
+                
+            {
+                viewModel.CanUserEditUkprn = true;
+            }
+            else if(viewModel.GroupType == GT.SecureSingleAcademyTrust
+                && User.InRole(EdubaseRoles.ROLE_BACKOFFICE + "," + EdubaseRoles.UKRLP))
             {
```

Comment:
Discsussed with @anthony9853 - use methods to reduce if and else statements in this section

eg.
public bool Can...(...)
{
   if (conditions1) {return true;}
...
    return false;
}